clouddriver-openstack/src/main/groovy/com/netflix/spinnaker/clouddriver/openstack/deploy/description/servergroup/ServerGroupParameters.groovy

```diff
@@ -106,7 +109,8 @@ class ServerGroupParameters {
       rawUserData: params.get('user_data'),
       tags: unescapePythonUnicodeJsonMap(params.get('tags') ?: null),
       sourceUserDataType: params.get('source_user_data_type'),
-      sourceUserData: params.get('source_user_data')
+      sourceUserData: params.get('source_user_data'),
+      asgResourceFilename: params.get('asg_resource_filename')
```

Comment:
As discussed offline, if the OpenStack provider is intended to also be used for non-aws platforms, we should try to avoid introducing more `asg` named entities.
No need to do that as part of this PR though.